demo/src/test/java/amazon/SearchFieldTest.java

```diff
@@ -0,0 +1,137 @@
+package amazon;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SearchFieldTest {
+   protected WebDriver driver;
+   protected String searchData = "laptop"; //default
+
+   public SearchFieldTest() {    
+      // before before
+      System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
+      this.driver = new ChromeDriver();
+      driver.manage().window().maximize();
+      driver.manage().timeouts().pageLoadTimeout(Duration.ofSeconds(10));
+      driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+   }
+
+   @BeforeMethod
+   public void before(){
+      driver.get("https://www.amazon.com/");
+      PageFactory.initElements(driver, this);
+   }
+
+   @AfterTest
+   public void afterTest() {
+      driver.close();
+      driver.quit();
+   }
+
+   @FindBy(id = "twotabsearchtextbox")
+   private WebElement searchField;
+
+   @FindBy(xpath = "//span[@data-component-type='s-search-results']//span[@class='a-size-medium a-color-base']")
+   private List<WebElement> searchNoResultElementS;   
+
+   @FindBy(xpath = "//span[@data-component-type='s-result-info-bar']//div[@class='a-section a-spacing-small a-spacing-top-small']//span")
+   private List<WebElement> searchResults;
+
+   @FindBy(xpath = "//div[@data-component-type='s-search-result']//span[@class='a-size-medium a-color-base a-text-normal']")
+   private List<WebElement> searchCardsProduct;
+
+
+   @DataProvider(name = "searchData")
+      public Object[][] searchData() {
+         return new Object[][] {
+                  {"laptop"},
+                  // {"phone"},
```

Comment:
Why did you comment on these lines?

Reply:
In task we have one input data. And for faster verification of the verifier.